# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../a2ps/glibc-2.3.4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some invalid assumptions that fail since glibc-2.3.4.

  - Rene Rebe <rene@exactcode.de>

--- a2ps-4.13/lib/path-concat.c.vanilla	2005-01-06 19:08:32.555111856 +0100
+++ a2ps-4.13/lib/path-concat.c	2005-01-06 19:08:36.959442296 +0100
@@ -31,8 +31,6 @@
 #endif
 #include <sys/types.h>
 
-char *malloc ();
-
 #ifndef DIRECTORY_SEPARATOR
 # define DIRECTORY_SEPARATOR '/'
 #endif
